### PR TITLE
variable "source_db" , for rds replica option

### PR DESCRIPTION
**Pull request by agomezvidalee**
Allow origin database for RDS replica.

### CODE REVIEW

File: modules/db_instance/main.tf [hunk 1]
```diff
@@ -17,6 +17,8 @@ resource "aws_db_instance" "this" {
   password = "${var.password}"
   port     = "${var.port}"
 
+  replicate_source_db = "${var.source_db}"
```
Comment (antonbabenko) [inline modules/db_instance/main.tf]: use variable and value the same - "replicate_source_db" (not "source_db")

File: modules/db_instance/variables.tf [hunk 2]
```diff
@@ -21,6 +21,11 @@ variable "kms_key_id" {
   default = ""
 }
 
+variable "source_db" {
+  description = "The master DB for replica"
```
Comment (antonbabenko) [inline modules/db_instance/variables.tf]: Make description more informative (follow [official documentation](https://www.terraform.io/docs/providers/aws/r/db_instance.html#replicate_source_db)):
> Specifies that this resource is a Replicate database, and to use this value as the source database. This correlates to the identifier of another Amazon RDS Database to replicate.

File: modules/db_instance/main.tf (hunk 1 above)
Comment (agomezvidalee) [reply modules/db_instance/main.tf]: updated: replicate_source_db = "${var.replicate_source_db}"

File: modules/db_instance/variables.tf (hunk 2 above)
Comment (agomezvidalee) [reply modules/db_instance/variables.tf]: description improved

**Comment by agomezvidalee** [comment]
Last changes commented in-line.

File: examples/complete/main.tf [hunk 3]
```diff
@@ -44,6 +44,9 @@ module "db" {
   backup_window           = "03:00-06:00"
   backup_retention_period = 0                     // disable backups to create DB faster
 
+  # On replica option - Source DB for RDS replica
+  replicate_source_db = "${var.replicate_source_db}"
```
Comment (antonbabenko) [inline examples/complete/main.tf]: Let's remove this from example, because it won't be used so often.

**Comment by antonbabenko** [comment]
Rebase this PR, please.

**Comment by antonbabenko** [comment]
Thanks @agomezvidalee ! Your code has been released as [v1.1.1](https://github.com/terraform-aws-modules/terraform-aws-rds/releases/tag/v1.1.1).

**Pull request merged**